README.md

```diff
@@ -93,19 +93,33 @@ $ COSMOS_NODE=localhost yarn testnet
 ---
 
 ### Production
-Get the Gaia binary from [GitHub](`https://github.com/cosmos/gaia/releases`).
 
-Install [Docker](https://docs.docker.com/get-started/).
+Building requires that [Docker](https://www.docker.com/get-docker) is installed
+on your system.
 
-Build and run the app.
+Build the app:
+
+The first argument is a commit for Voyager (e.g., the tag `0.4.3`).  Optional
+arguments (such as `--platform`) must come afterwards.
```

Comment:
if `--platform` is optional, then we need to detect the OS of the dev I think. I would prefer making it mandatory so the dev knows for what he is building. The build process will in most cases be cross-platform I guess.

Reply:
If `--platform` isn't specified then it uses the default that was already in the code: `darwin,linux,win32`

Reply:
It is not possible to have a default platform. The binary is OS specific. I think it is a mistake it is still in there.

Reply:
Right, that makes sense.